app/components/UpcomingEvent/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { PropTypes } from 'react';
+
+const UpcomingEvent = ({ event, index }) => (
+  <article className="UpcomingEvent" key={index}>
```

Comment:
Change the props of this component so that instead of using it like this:

```html
<UpcomingEvent event={event} index={index} />
```
You use it like this
```html
<UpcomingEvent {...event} key={index} />
```

The props would then be `({ title, date, venue, startTime, endTime })`